lib/pbench/server/api/__init__.py

```diff
@@ -167,7 +166,7 @@ def register_endpoints(api, app, config):
         UserAPI,
         f"{base_uri}/user/<string:target_username>",
         endpoint="user",
-        resource_class_args=(logger, token_auth),
+        resource_class_args=(logger,),
```

Comment:
If you make an edit for some other reason, there's an extra comma (or two) in this line.

Reply:
I'll just pre-reconcile this....

Reply:
Well, that extra comma inside is required because `resource_class_args` requires a tuple and without a comma, it will not a be type tuple with just one value.

Reply:
Ah, excellent!

(I curse Black for making me thing otherwise.)